massa-async-pool/Cargo.toml

```diff
@@ -19,7 +19,7 @@ massa_serialization = { path = "../massa-serialization" }
 massa_signature = { path = "../massa-signature" }
 massa_db_exports = { path = "../massa-db-exports" }
 massa_time = { path = "../massa-time" }
-massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs", rev = "18ec02f", features = ["tonic"] }
+massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs", rev = "ba569c8", features = ["tonic"] }
```

Comment:
I feel like we must merge massa-proto#feature/Improve_ABI_types_in_wasmv1 into massa-proto#main in order for this commit sha to be correct